src/api/v1/types.ts

```diff
@@ -445,3 +448,60 @@ export class GetTableByScopeResponse extends Struct {
     @Struct.field(GetTableByScopeResponseRow, {array: true}) rows!: GetTableByScopeResponseRow[]
     @Struct.field('string') more!: string
 }
+
+export interface OrderedActionsResult {
+    global_action_sequence: UInt64
+    account_action_sequence: Int32,
```

Comment:
Should be a 64 since our APIs return them 👍 

(This was what we talked about this on our dev call)